### PR TITLE
[RELEASE-1.7][BACKPORT] Allow setting seccompProfile to enable using …

### PR DESCRIPTION
**Pull request by ReToCode**
Cherry-pick of https://github.com/openshift-knative/serving/pull/9

### CODE REVIEW

**Comment by skonto** [comment]
```
  activator_test.go:133: Did not observe activator-764744bb7d-6qv2f to actually be deleted: pod "activator-764744bb7d-6qv2f" is not in desired state, got: <*>&Pod{ObjectMeta:{activator-764744bb7d-6qv2f activator-764744bb7d- knative-serving  dbf1bbbe-9905-4dfd-a5fa-c0d36041a145 96663 0 2022-12-21 08:07:06 +0000 UTC 2022-12-21 09:01:50 +0000 UTC 0xc0008cb348 map[app:activator app.kubernetes.io/component:activator app.kubernetes.io/name:knative-serving app.kubernetes.io/version:v1.2.0 pod-template-hash:764744bb7d role:activator] map[cluster-autoscaler.kubernetes.io/safe-to-evict:false 
```

**Comment by skonto** [comment]
Last PR on this branch that was passing is this one: https://github.com/openshift-knative/serving/pull/69 (8/12).

File: config/core/configmaps/features.yaml [hunk 1]
```diff
@@ -109,6 +109,7 @@ data:
     # - RunAsNonRoot
     # - SupplementalGroups
     # - RunAsUser
+    # - SeccompProfile
```
Comment (nak3) [inline config/core/configmaps/features.yaml:112]: Just leaving a note as a FYI:
This yaml file is not used but https://github.com/openshift-knative/serving/blob/release-v1.7/openshift/release/artifacts/2-serving-core.yaml#L5000-L5006 is used on 1.7 branch (release-next & main fixed).

This is a comment line change so I think we don't need to update it, though.

Comment (ReToCode) [reply config/core/configmaps/features.yaml:112]: Hm I see, was it used in 1.6 (https://github.com/openshift-knative/serving/pull/9/files)? Maybe @skonto just added that for completeness sake?

Comment (skonto) [reply config/core/configmaps/features.yaml:112]: Yes that was meant for completeness back then.

**Pull request merged**